lib/google_maps.ex

```diff
@@ -851,6 +851,197 @@ defmodule GoogleMaps do
     GoogleMaps.get("place/queryautocomplete", params)
   end
 
+@doc """
+  Search for nearby places based on location and radius.
+
+  The Google Places API Web Service allows you to query
+  for place information on a variety of categories, 
+  such as: establishments, prominent points of interest,
+  geographic locations, and more. You can search for places
+  either by proximity or a text string. A Place Search 
+  returns a list of places along with summary information
+  about each place; additional information is available
+  via a Place Details query
+
+  ## Args:
+
+    * `location` — The latitude/longitude around which to
```

Comment:
If you decide to support lat/lon tuple besides string, can you update this argument's description?